surface/surface/urls.py

```diff
@@ -22,4 +22,4 @@
     path("", admin.site.urls),
     path("dkron/", include("dkron.urls")),
     path("sbomrepo/", include("sbomrepo.urls")),
-]
+]
```

Comment:
I've been noticing files not ending with new-lines. It's our convention that they do - please configure your editor accordingly when submitting your changes.